fed/mosaic-application/src/main/java/org/eclipse/mosaic/fed/application/app/api/os/TrafficManagementCenterOperatingSystem.java

```diff
@@ -66,6 +66,14 @@ public interface TrafficManagementCenterOperatingSystem extends ServerOperatingS
      */
     ChangeLaneState changeLaneState(String edge, int laneIndex);
 
+    /**
+     * Provides a facility to change the lane state of a Variable Message Sign (VMS).
+     *
+     * @param signId      The ID of the sign.
+     * @param laneIndex   The index of the lane (0 = right-most lane).
+     */
+    ChangeLaneState changeVmsLaneState(String signId, int laneIndex);
```

Comment:
I would just call it `changeVmsState` or `changeVariableMessageSignState`, but keep the name of the return type.